src/app/project/controllers/managements.ctrl.js

```diff
@@ -184,6 +184,12 @@ angular.module('app.project').controller('ManagementsCtrl', [
     };
 
     $scope.tableControl.recordsNotFiltered = function() {
+      if (
+        $scope.tableControl.records &&
+        $scope.tableControl.records.length > managementRecordsCount
```

Comment:
Should this be `$scope.tableControl.records.length != managementRecordsCount`? (Same for sites)